examples/dev/slicing/profile_points.py

```diff
@@ -23,7 +23,7 @@
 
 # Profiling
 with cProfile.Profile() as pr:
-    for k in range(reps):
+    for _k in range(reps):
```

Comment:
Just 
```suggestion
    for _ in range(reps):
```

If k is unused ?

Reply:
ruff just add `_` to not lose context stored in the variable name, but in this case, no context is stored.